### PR TITLE
Amplitude: track datasource updated

### PR DESCRIPTION
**Pull request by PopDaph**
## Description

Adds a new DataSource updated event to Amplitude. 
Context: https://github.com/dust-tt/tasks/issues/583

I duplicated the code from DataSourceCreated instead of trying to factorize to keep flat tracking code.

### CODE REVIEW

File: front/lib/amplitude/back/generated/index.ts [hunk 1]
```diff
@@ -98,6 +98,13 @@ export interface DataSourceCreatedProperties {
   workspaceId: string;
   workspaceName: string;
 }
+export interface DataSourceUpdatedProperties {
```
Comment (lasryaric) [inline front/lib/amplitude/back/generated/index.ts:101]: @PopDaph did you write this code or did you generate it from Amplitude/ampli ?

**Review by lasryaric — COMMENTED** [review]
Left a first question

File: front/lib/amplitude/back/generated/index.ts (hunk 1 above)
Comment (PopDaph) [reply front/lib/amplitude/back/generated/index.ts:101]: I wrote it 😬

Comment (lasryaric) [reply front/lib/amplitude/back/generated/index.ts:101]: @PopDaph that's on me, I should have documented it.
I will invite you to Amplitude and write a little doc.

**Comment by lasryaric** [comment]
@PopDaph documentation is here, along with the new script that you just approved:
https://www.notion.so/dust-tt/Runbook-Local-Dev-setup-720bffc7d67a447184cb957e9471380c

**Comment by PopDaph** [comment]
Thanks @lasryaric , closing this PR will reopen a new one with the valid process 🙇🏻‍♀️

**Pull request closed without merging**